onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/layer_norm_fusion.h"
+#include "core/graph/graph_utils.h"
+#include "float.h"
+#include <deque>
+
+using namespace ONNX_NAMESPACE;
+using namespace onnxruntime::common;
+namespace onnxruntime {
+
+static const Node* first_child_by_type(Node& node, const std::string& child_type) {
```

Comment:
>first_child_by_type [](start = 19, length = 19)

Please follow the function naming rule

Reply:
And you can move those 2 functions to graph_utils.h. So other fusion can leverage it.

---
In reply to: [343271032](https://github.com/microsoft/onnxruntime/pull/2319#discussion_r343271032) [](ancestors = 343271032)